python/pyspark/ml/clustering.py

```diff
@@ -271,18 +269,18 @@ class GaussianMixture(JavaEstimator, _GaussianMixtureParams, JavaMLWritable, Jav
     >>> gaussians[0].mean
     DenseVector([0.825, 0.8675])
     >>> gaussians[0].cov.toArray()
-    array([[ 0.005625  , -0.0050625 ],
-           [-0.0050625 ,  0.00455625]])
+    array([[ 0.00562..., -0.00506...],
```

Comment:
How much did the answer change? trying to figure out whether it's more or less accurate. Unless the perf difference is significant I wouldn't want to lose much here.